doc/build-instructions/Build_Instructions_V13.md

```diff
@@ -0,0 +1,654 @@
+```
+
+  - For Linux on 64-bit Power systems, run:
+```
+wget https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/jdk13/ppc64le/ubuntu16/Dockerfile
+```
+
+  - For Linux on 64-bit z Systems, run:
+```
+wget https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/jdk13/s390x/ubuntu16/Dockerfile
+```
+
+3. Next, run the following command to build a Docker image, called **openj9**:
+```
+docker build -t openj9 -f Dockerfile .
+```
+
+4. Start a Docker container from the **openj9** image with the following command, where `-v` maps any directory, `<host_directory>`,
+on your local system to the containers `/root/hostdir` directory so that you can store the binaries, once they are built:
+```
+docker run -v <host_directory>:/root/hostdir -it openj9
+```
+
+:pencil: Depending on your [Docker system configuration](https://docs.docker.com/engine/reference/commandline/cli/#description), you might need to prefix the `docker` commands with `sudo`.
+
+Now that you have the Docker image running, you are ready to move to the next step, [Get the source](#2-get-the-source).
+
+#### Setting up your build environment without Docker
+
+If you don't want to user Docker, you can still build an **OpenJDK V13** with OpenJ9 directly on your Ubuntu system or in a Ubuntu virtual machine. Use the
+[Linux on x86 Dockerfile](https://github.com/eclipse/openj9/blob/master/buildenv/docker/jdk13/x86_64/ubuntu16/Dockerfile) like a recipe card to determine the software dependencies
+that must be installed on the system, plus a few configuration steps.
+
+:pencil:
+Not on x86? We also have Dockerfiles for the following Linux architectures: [Linux on Power systems](https://github.com/eclipse/openj9/blob/master/buildenv/docker/jdk13/ppc64le/ubuntu16/Dockerfile) and [Linux on z Systems](https://github.com/eclipse/openj9/blob/master/buildenv/docker/jdk13/s390x/ubuntu16/Dockerfile).
+
+1. Install the list of dependencies that can be obtained with the `apt-get` command from the following section of the Dockerfile:
+```
+apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    gcc-7 \
+    g++-7 \
+    autoconf \
+    ca-certificates \
+    ...
+```
+
+2. This build uses the same gcc and g++ compiler levels as OpenJDK, which might be
+backlevel compared with the versions you use on your system. Create links for
+the compilers with the following commands:
+```
+ln -s g++ /usr/bin/c++
+ln -s g++-7 /usr/bin/g++
+ln -s gcc /usr/bin/cc
+ln -s gcc-7 /usr/bin/gcc
+```
+
+3. Download and setup **freemarker.jar** into a directory.
+```
+cd /<my_home_dir>
+wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
+tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
+rm -f freemarker.tgz
+```
+
+4. Download and setup the boot JDK using the latest AdoptOpenJDK v11 build.
```

Comment:
These instructions still refer to jdk11, but you need jdk12 or jdk13 to build jdk13.